packages/google_workspace/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.2.4"
```

```suggestion
- version: "2.3.0"
```
since this is an enhancement.

---

thanks! made a new commit with this change.